packages/wallets/src/components/WalletCurrencyIcon/WalletCurrencyIcon.scss

```diff
@@ -0,0 +1,13 @@
+.wallets-currency-icon {
+    width: 7rem;
+    height: fit-content;
+
+    & > svg {
+        width: auto;
+        height: 3.5rem;
+
```

Comment:
Explicitly set the width here, there was an issue with the SVG scaling in particular with the cryptocurrency icons:
Before:
<img width="518" alt="Screenshot 2023-09-18 at 2 45 30 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/be397b91-1129-4781-9493-05dadfea6189">

After:
<img width="427" alt="Screenshot 2023-09-18 at 2 45 45 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/35ec70a3-8e8b-4c86-adb1-deba9fa29950">